src/components/TeaserFront/Text.js

```diff
@@ -92,18 +92,28 @@ const Text = ({
   position,
   center,
   color,
+  collapsedColor,
   maxWidth,
   margin
 }) => {
   const textAlign = center ? 'center' : ''
   const rootStyles = position ? styles.rootPosition : {}
   const middleStyles = position === 'middle' ? styles.rootMiddle : {}
+
+  const colorStyle = collapsedColor && css({
+    color: collapsedColor,
+    [tUp]: {
+      color
+    }
+  })
+
   return (
     <div {...rootStyles} {...middleStyles}>
       <div
         {...attributes}
+        {...colorStyle}
         {...css(styles.positioned, position ? styles[position] : {})}
-        style={{ color, textAlign, maxWidth, margin }}
+        style={{ color: !collapsedColor && color, textAlign, maxWidth, margin }}
```

Comment:
This may result in `color:;text-align:` (`style={{color: false, textAlign: ''}}`). Use `undefined` instead of `false` and `''` (`style={{color: undefined, textAlign: undefined}}` results in no style attribute at all).